src/server/heart-beat-db.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const MongoClient = require('mongodb').MongoClient;
+const url = 'mongodb://localhost:27017/heartbeat';
+
+const checkDatabaseHealth = function(res) {
+  MongoClient.connect(url, function(err, db) {
+    if (err === null) {
+      let collection = db.collection('heartbeat');
+      collection.find({}).toArray(function(err, docs) {
+        if (err === null && docs.length>0) {
+          res.json({"status": "ok", "database": "ok"});
```

Comment:
put the objects into const variables to the top, we dont like magic numbers and strings. also you decrease code duplication as well. use single quotes